jb/fftw/tde_result.hpp

```diff
@@ -0,0 +1,122 @@
+#ifndef jb_fftw_tde_result_hpp
+#define jb_fftw_tde_result_hpp
+
+#include <jb/detail/array_traits.hpp>
+#include <jb/fftw/aligned_multi_array.hpp>
+
+namespace jb {
+namespace fftw {
+
+/**
+ * tde_result is a type to handle the result of operating over the
+ * last dimension array of a multi array of timeseries.
+ * This is used when the operation returns a value out of the array,
+ * therefore tde_result has one dimension less than the
+ * tde_result constructor's argument.
+ */
+
+/**
```

Comment:
I do not think you can break the comments across two doxygen blocks.  You should move the longer description to the lower block.  Typically the form of a doxygen block is:

```
/**
  * This is a short description of the thing, should fit in a line.
  *
  * This is a long description of the thing, wax poetically about it.
  *
  * @tparam XYZ parameters and template parameters described
  * @return what does it return if it is a function.
  * @throws what it might throw if it is a function.
  */
```